### PR TITLE
Upgrade

### PR DESCRIPTION
**Pull request by nemchik**
- Use the `mariadb` specific commands (ex: `mysqladmin` is a symlink to `mariadb-admin`)
- Document information about upgrading the database using `mariadb-upgrade`
- Add logic after `svc-mariadb` to check if an upgrade is required and notify the user in logs

### CODE REVIEW

**Review by thespad — CHANGES_REQUESTED** [review]
image

**Comment by nemchik** [comment]
> image

Reworked to have the upgrade check run after `svc-mariadb` is ready. It is also now completely silent unless an upgrade is needed (and the `MYSQL_ROOT_PASSWORD` variable contains a valid root password).

File: root/etc/s6-overlay/s6-rc.d/init-mariadb-upgrade/run [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# display a message about upgrading database if needed
+if mariadb-upgrade -u root -p"${MYSQL_ROOT_PASSWORD}" --check-if-upgrade-is-needed >/dev/null 2>&1; then
+    cat <<-EOF
+
+
+
+#################################################################
+#   An upgrade is required on your databases. Run the command   #
+#             mariadb-upgrade -u root -p<PASSWORD>              #
+#################################################################
```
Comment (thespad) [inline root/etc/s6-overlay/s6-rc.d/init-mariadb-upgrade/run:14]: I would suggest rewording this to include something like "Stop any service that are accessing databases in this container. and then run the command..."

Comment (nemchik) [reply root/etc/s6-overlay/s6-rc.d/init-mariadb-upgrade/run]: Resolved in https://github.com/linuxserver/docker-mariadb/pull/112/commits/ad6f6c57af7a7539d3831891fe026b3d2f20b916

**Review by thespad — APPROVED** [review]
